### PR TITLE
Fix predicate for initializing caches. Remove depricated code which r…

### PR DESCRIPTION
**Pull request by PhobicHD**
…unelite client removed/changed. Add binds and properties needed to inject into the client.

### CODE REVIEW

**Comment by PhobicHD** [comment]
I think I broke review it but don't pull it

File: src/main/java/net/runelite/client/modified/BotModule.java
Comment (GigiaJ) [inline src/main/java/net/runelite/client/modified/BotModule.java:1]: Nice, just some updates to reflect whatever changes RuneLite has made.

File: src/main/java/net/runelite/rsb/botLauncher/Application.java
Comment (GigiaJ) [inline src/main/java/net/runelite/rsb/botLauncher/Application.java:1]: I think the ! should be outside the parenthesis for the aesthetic, but that is alright. :+1:

**Review by GigiaJ — APPROVED** [review]
Just some needed changes removing deprecated features and getting stuff functional again. Nice.
For future PRs I'd encourage breaking changes up a little better in commits, but I happen to do slightly sweeping changes myself. Nice work mate.

**Pull request merged**